src/main/java/com/redhat/devtools/intellij/lsp4mp4ij/settings/MicroProfileInspectionsInfo.java

```diff
@@ -52,6 +61,25 @@ public static MicroProfileInspectionsInfo getMicroProfileInspectionInfo(Project
         wrapper.valueSeverity = SeverityMapping.getSeverity(MicroProfilePropertiesValueInspection.ID, profile);
         wrapper.requiredSeverity = SeverityMapping.getSeverity(MicroProfilePropertiesRequiredInspection.ID, profile);
         wrapper.expressionSeverity = SeverityMapping.getSeverity(MicroProfilePropertiesExpressionsInspection.ID, profile);
+        wrapper.unassignedSeverity = SeverityMapping.getSeverity(MicroProfilePropertiesUnassignedInspection.ID, profile);
+
+        InspectionToolWrapper<?, ?> toolWrapper = profile.getInspectionTool(MicroProfilePropertiesUnknownInspection.ID, project);
+        if (toolWrapper != null) {
```

Comment:
By using AbstractDelegateInspectionWithExcludedProperties I think you can write a method fillExcludeList method to avoid duplicate code.

Reply:
done